tee-worker/client-api/parachain-api/prepare-build/interfaces/sidechain/definitions.ts

```diff
@@ -86,7 +86,6 @@ export default {
                 UnexpectedMessage: "Null",
                 __Unused_WrongSignatureType: "Null",
                 VerifyWeb3SignatureFailed: "Null",
-                NoEligibleIdentity: "Null",
```

Comment:
@jonalvarezz we intentionally removed it (instead of leaving a placeholder): https://github.com/litentry/litentry-parachain/pull/2739#discussion_r1605571350

Reply:
Thanks @Kailai-Wang, I don't understand why the example in the other comment is wrong. Isn't it how we have been managing enum changes? Or is it because in this case, it is the last item of the enum?

Reply:
> Or is it because in this case, it is the last item of the enum?

Exactly :)

Reply:
And since you remove the entry in rust enum, it's better to do it consistently in ts